__tests__/tests/components/ToggleView.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Text } from 'react-native';
+import sinon from 'sinon';
+import { shallow, render } from 'enzyme';
+
+import { ToggleView } from 'components';
+
+describe('<ToggleView />', () => {
+  it('correctly renders with default touchable view', () => {
+    const wrapper = render(
+      <ToggleView>
+        <Text>I'm hidden</Text>
```

Comment:
GitHub doesn't like these quotation marks for some reason (though I believe they'll compile fine). Maybe just replace with "I am hidden"?